microsoft/testsuites/performance/storageperf.py

```diff
@@ -455,13 +464,34 @@ def _perf_premium_datadisks(
         max_iodepth: int = 256,
     ) -> None:
         disk = node.features[Disk]
-        data_disks = disk.get_raw_data_disks()
-        disk_count = len(data_disks)
-        assert_that(disk_count).described_as(
-            "At least 1 data disk for fio testing."
-        ).is_greater_than(0)
-        partition_disks = reset_partitions(node, data_disks)
-        filename = ":".join(partition_disks)
+        if disk.is_asap_enabled :
```

Comment:
Should it be added as another test case? This method called "datadisks". It's confused to test with os disk.

Reply:
We are not testing OS disk here. We are just checking OS info for detecting the disk controller type. The test is still about data disks only.